qiskit/transpiler/preset_passmanagers/default.py

```diff
@@ -72,7 +72,9 @@ def default_pass_manager(transpile_config):
     pass_manager.append(Decompose(SwapGate))
 
     # Change CX directions
-    pass_manager.append(CXDirection(coupling_map))
+    pass_manager.append(CXDirection(coupling_map),
+                        condition=lambda property_set: (not coupling_map.is_symmetric and
+                                                        not property_set['is_direction_mapped']))
```

Comment:
Should we want to add a `CheckCXDirection` pass prior?

Reply:
Yes... also `CheckCXDirection` should only run when `not coupling_map.is_symmetric`. I will do that in a following PR (it is not directly related)...

Reply:
Summitted in https://github.com/Qiskit/qiskit-terra/issues/2362